### PR TITLE
tikv: add raftstore.report-min-resolved-ts-interval

### PR DESCRIPTION
**Pull request by Oreoxmt**
Signed-off-by: Aolin <aolin.zhang@pingcap.com>

### What is changed, added or deleted? (Required)

Add a configuration item `raftstore.report-min-resolved-ts-interval` in TiKV configuration file.

`raftstore.report-min-resolved-ts-interval` is introduced in https://github.com/tikv/tikv/pull/12157 (on Mar 17, 2022) and the item applies to TiDB v6.1 and later versions.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/pull/12157

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lhy1024 — COMMENTED** [review]
Shall we provide doc in Chinese also?

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -964,6 +964,12 @@ Configuration items related to Raftstore.
 + Default value: `1MB`
 + Minimum value: `0`
 
+### `report-min-resolved-ts-interval`
+
++ Determines the minimum interval at which the resolved timestamp is reported to the PD leader. If this value is set to `0`, it means that the reporting is disabled.
++ Default value: `"1s"`
```
Comment (lhy1024) [inline tikv-configuration-file.md:970]: The unit of this configuration is also seconds, so the smallest positive value is also 1s

**Review by lhy1024 — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -964,6 +964,12 @@ Configuration items related to Raftstore.
...
++ Minimum value: `0`
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:971]:
```suggestion
+ Minimum value: `1s`
```

File: tikv-configuration-file.md (hunk 1 above)
Comment (Oreoxmt) [inline tikv-configuration-file.md:970]:
```suggestion
+ Default value: `"1s"`, which is the smallest positive value
```

**Review by lhy1024 — APPROVED** [review]

**Comment by Oreoxmt** [comment]
> Shall we provide doc in Chinese also?

@lhy1024 Yes, I will translate it to Chinese after this PR is merged

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -964,6 +964,12 @@ Configuration items related to Raftstore.
 + Default value: `1MB`
 + Minimum value: `0`
 
+### `report-min-resolved-ts-interval`
+
++ Determines the minimum interval at which the resolved timestamp is reported to the PD leader. If this value is set to `0`, it means that the reporting is disabled.
++ Default value: `"1s"`, which is the smallest positive value
++ Minimum value: `0`
```
Comment (shichun-0415) [inline tikv-configuration-file.md:971]:
```suggestion
+ Minimum value: `0`
+ Unit: second
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**